ztp/ztp-ran-policy-generator/README.md

```diff
@@ -1,11 +1,265 @@
-# ztp-ran-policy-generator
+# Ran Policy Generator (RanGen)
+
+The Ran policy generator (RanGen) is a kustomize plugin used to facilitate creating ACM policies from a predefined Custom Resources (CRs). There are 3 main items that RanGen rely on to generate the ACM policies and its placement binding and placement rule.
+
+1 - Policy Categorization:
+
+ Any generated policy needs to be placed under one of the following categories; 
+
+   - Common: a policy exist under the common category will be applied to all clusters.
+
+   - Groups: a policy exist under the group category will be applied to group of clusters.
+
+   - Sites: a policy exist under the sites category will be applied to a specific cluster.
```

Comment:
Aren't we mixing between the tool and the labeling scheme? I mean that `Common`, `Groups` and `Sites` is our scheme for applying manifests to clusters that minimizes duplication and simplifies the structure by grouping. But does it have anything to do with the tool?
Maybe, if we want to describe this scheme, it's possible to add a separate readme file in the examples folder

Reply:
the categorization is a concept in the tool, and the policyGen template required that to locate the generated policy.